packages/wallet/backend/src/rafiki/service.ts

```diff
@@ -331,7 +364,18 @@ export class RafikiService implements IRafikiService {
   }
 
   private async handleOutgoingPaymentFailed(wh: WebHook) {
-    const source_ewallet = await this.getRapydWalletIdFromWebHook(wh)
+    const paymentPointer = await this.getPaymentPointer(wh)
+
+    if (paymentPointer.isWM) {
+      await this.deps.wmTransactionService.updateTransaction(
+        { paymentId: wh.data.payment.id },
+        { status: 'FAILED', value: 0n }
```

Comment:
I'm a bit unsure about having `0n` as a value. Even if the outgoing payment fails, there might still be an amount sent.

Reply:
I don't think this is the case